subprojects/docs/src/docs/userguide/jvm/java_testing.adoc

```diff
@@ -282,6 +282,11 @@ include::sample[dir="snippets/testing/junit-xml-results/kotlin",files="build.gra
 include::sample[dir="snippets/testing/junit-xml-results/groovy",files="build.gradle[tags=configure-content]"]
 ====
 
+===== omitSystemOutLog & omitSystemErrLog
```

Comment:
Please update to use the new property names.  Also mention that the default _remains_ to continue to include both logs' output.